html/pages/device/edit/ports.inc.php

```diff
@@ -1,10 +1,12 @@
+</form>
 <span id="message"><small><div class="alert alert-danger">n.b For the first time, please click any button twice.</div></small></span>
```

Comment:
That's from another PR not this one - I can't get the Javascript to fire the first time for clicking the all buttons so had to put that in so people were aware.

Unrelated to this PR though :)